Eastern-Research-Group/mywaterway
# Feature/state page accessibility

I used axe, lighthouse, wave and html_codesniffer to test the accessibility. There is quite a bit of noise with these extensions, due to the EPA One Template and the glossary panel. 

I ignored some items that were flagged by wave.  
1. 1 X Missing fieldset - None of the other extensions flagged this one. I don't think we really need to add this to the radio buttons.
2. 11 X Redundant title text - We are doing this to display button tool tips on small screens and esri is doing the same thing. 

## Related Issues:
* [https://app.breeze.pm/projects/100762/cards/3333226](https://app.breeze.pm/projects/100762/cards/3333226)

## Main Changes:
* Updated the styles for react-select components to fix contrast errors. I think I got all of the react-select components, without touching the community tabs. 
* Fixed label/input linkage accessibility issues.
* Fixed issues with skipping over headings. 
    * The styles changed slightly for the "Choose a Topic" heading on the water quality overview tab.
* Fixed issues with extensions saying the `<p>` should be headings for the StyledNumber component. 

## Steps To Test:
1. Go to the state page 
2. Run wave and check issues
3. Run axe and check issues
4. Run lighthouse and check issues
5. Select Kansas
6. Click Eating Fish and select "Rivers and Streams" from the "Water Type" drop down
7. Redo steps 2 - 4 
8. Go to the advanced search tab and click search
9. Redo steps 2 - 4
10. Click the list view
11. Redo steps 2 - 4

## TODO:
- [ ] Fix accessibility issues with the EPA One Template
- [ ] Fix accessibility issues with the glossary panel. I think we will be able to do this without touching the forked glossary component 🤞 .
